modules/ROOT/pages/creating-an-odatav2-api-with-apikit.adoc

```diff
@@ -27,15 +25,15 @@ The following example shows you how to implement an OData-enabled REST API:
 . Copy the `odata.raml` file you downloaded in the prerequisites section to the `/src/main/resources/api` directory in your project.
 . In the project explorer, right-click `odata.raml` and select *Mule* > *Generate OData API from RAML Types*.
 +
-image::generate-odata.png["Window highlights the full path that was previously mentioned to get to the generate OData API from RAML types option."]
+image::generate-odata.png["A window highlighting the full path to the generate OData API from RAML types option"]
 . The OData extension generates the necessary files for your OData API implementation:
 +
 image::list-of-odata-files.png["The three files are highlighted in the tree: api.xml in the Mule folder, and api.raml and odatalibrary.raml files in the resource folder."]
 
 [calloutlist]
-. The Mule configuration file with the flows for the API implementation
-. The generated RAML definition
-. The generated RAML libraries
+.. The Mule configuration file with the flows for the API implementation
```

Comment:
For brevity, remove the leading The in this list.

Reply:
OK! Thanks!